docs/user-guides/eks.md

```diff
@@ -13,7 +13,7 @@ tEKS uses launch template by default and use one node pool per availability zone
 You can use any inputs from the upstream module to configure the cluster in `eks/terragrunt.hcl`.
 
 ```json
-{!terraform/live/demo/eu-west-3/eks/terragrunt.hcl!}
+{!terragrunt/live/demo/eu-west-3/clusters/full/eks/terragrunt.hcl!}
 ```
 
 ## Customizations
```

Comment:
all the psp part can be removed are they are not used anymore and are deprecated

Reply:
I removed the whole Customizations part. :+1: